M/MuJoCo/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "MuJoCo"
+version = v"2.3.7"
+
+# Collection of sources required to build libX11
+sources = [
+    GitSource("https://github.com/deepmind/mujoco", "c9246e1f5006379d599e0bcddf159a8616d31441")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mujoco
+sed -i '/Werror/d' cmake/MujocoOptions.cmake
+CPPFLAGS="-I${prefix}/include"
+CXXFLAGS="-I${prefix}/include"
```

Comment:
Hmm, I thought the compiler automatically searched in these directories anyway? Why are they necessary here?

Reply:
It didn't compile without these unfortunately.